Minor suggestions, mostly punctuation-related (#40)
greenelab/mpmp-manuscript: content/03.methods.md

To generate binary mutated/non-mutated gene labels for our machine learning model, we used mutation calls for TCGA samples from MC3 [@doi:10.1016/j.cels.2018.03.002] and copy number threshold calls from GISTIC2.0 [@doi:10.1186/gb-2011-12-4-r41].
MC3 mutation calls were downloaded from the Genome Data Commons (GDC) of the National Cancer Institute, at [`https://gdc.cancer.gov/about-data/publications/pancanatlas`](https://gdc.cancer.gov/about-data/publications/pancanatlas).

> Copy number threshold calls are from an older version of PanCanAtlas, and are available here: [`https://figshare.com/articles/dataset/TCGA_PanCanAtlas_Copy_Number_Data/6144122`](https://figshare.com/articles/dataset/TCGA_PanCanAtlas_Copy_Number_Data/6144122).<!--Not sure what it is older relative to, becuase I don't think you've mentioned the phrase "PanCanAtlas" up until here!-->

Reviewer: ```suggestion
Copy number threshold calls are from an older version of the GDC data, and are available here: [`https://figshare.com/articles/dataset/TCGA_PanCanAtlas_Copy_Number_Data/6144122`](https://figshare.com/articles/dataset/TCGA_PanCanAtlas_Copy_Number_Data/6144122).
```

and the probability of no mutation in the target gene, $P(y^{*} = 0 \mid X^{*}; \hat{w})$, is given by (1 - the above quantity).

For each target gene, we evaluated model performance using two replicates of 4-fold cross-validation, where train and test splits were stratified by cancer type and sample type.
That is, each training set/test set combination had equal proportions of each cancer type (BRCA, SKCM, COAD, etc.) and each sample type (primary tumor, recurrent tumor, etc.).
To choose the elastic net hyperparameters, we used 3-fold nested cross-validation, with a grid search over the following hyperparameter ranges: $\lambda$ = [0.0, 0.05, 0.1, 0.3, 0.5, 0.7, 0.9, 1.0] and $\alpha$ = [0.0001, 0.001, 0.01, 0.1, 1, 10].
Using the grid search results, for each evaluation fold we selected the set of hyperparameters with the optimal area under the precision-recall curve (AUPR), averaged over the three inner folds.

### Evaluating mutation prediction classifiers

> Area under the receiver-operator curve (AUROC) [@doi:10.1016/j.patrec.2005.10.010] and the area under the precision-recall curve (AUPR) [@doi:10.1145/65943.65945] are metrics that are frequently used to quantify classification performance for a continuous or probabilistic output, such as that provided by logistic regression.

Reviewer: ```suggestion
Area under the receiver-operator curve (AUROC) [@doi:10.1016/j.patrec.2005.10.010] and area under the precision-recall curve (AUPR) [@doi:10.1145/65943.65945] are metrics that are frequently used to quantify classification performance for a continuous or probabilistic output, such as that provided by logistic regression.
```

To predict mutation presence or absence in cancer genes using multiple data types simultaneously, we concatenated individual datasets into a large feature matrix, then used the same elastic net logistic regression method described previously.
For this task, we considered only the gene expression, 27K methylation, and 450K methylation datasets.
We used only these data types to limit the number of multi-omics combinations; the expression and methylation datasets resulted in the best overall performance across the single-omics experiments, so we limited combinations to those datasets.

> In the main text, we report results using the top 5,000 principal components for each dataset to ensure that the information content of the included features is comparable between data types.<!--The top 5000 wouldn't necessarily capture the same amount of variance across different experiments, though, would they? It's more just that that is a lot of PCs so you're capturing quite a bit of variance?-->

Reviewer: ```suggestion
In the main text, we report results using the top 5,000 principal components for each dataset to ensure that most variance is captured (approximately 95-98% of variance for each data type).
```
I looked at this in greenelab/mpmp#19, and it does turn out that the amount of variance captured by 5000 PCs is about the same for expression and methylation (although not exactly the same). What you said is also true, though - we picked 5000 because it's likely that this many PCs will capture most of the useful signal.

Author: That definitely makes sense! I think it was the "to ensure" that was tripping me up (to me it implies that there's something I'm supposed to know about this). So I would think of what you are saying as something like "we report results using the top 5,000 principal components for each dataset, which ensures that most variance is captured (approximately 95-98% of variance for each data type)."

> In the main text, we report results using the top 5,000 principal components for each dataset to ensure that the information content of the included features is comparable between data types.<!--The top 5000 wouldn't necessarily capture the same amount of variance across different experiments, though, would they? It's more just that that is a lot of PCs so you're capturing quite a bit of variance?-->

Reviewer: ```suggestion
In the main text, we report results using the top 5,000 principal components for each dataset , which ensures that most variance is captured (approximately 95-98% of variance for each data type).
```
